tools/github_reporter/index.js

```diff
@@ -7410,7 +7410,7 @@ var require_client = __commonJS({
           throw new InvalidArgumentError("allowH2 must be a valid boolean value");
         }
         if (maxConcurrentStreams != null && (typeof maxConcurrentStreams !== "number" || maxConcurrentStreams < 1)) {
-          throw new InvalidArgumentError("maxConcurrentStreams must be a possitive integer, greater than 0");
+          throw new InvalidArgumentError("maxConcurrentStreams must be a positive integer, greater than 0");
```

Comment:
Changes to `github_reporter` should be upstreamed to https://github.com/MoLow/reporters

Reply:
Sure, not having much luck finding it though... https://github.com/search?q=repo%3AMoLow%2Freporters%20possitive&type=code

Reply:
CC @MoLow

Reply:
It's fixed upstream in Undici: <https://github.com/nodejs/undici/blob/0d00e468cb84b66d83635fa2920b19e3697c07dd/lib/dispatcher/client.js#L187>

Reply:
Done, upstreamed: https://github.com/nodejs/undici/pull/3640